xrpl-std/src/host/host_bindings.rs

```diff
@@ -713,6 +807,100 @@ unsafe extern "C" {
         out_buff_len: usize,
     ) -> i32;
 
+    /// Retrieves the issuer of a specific NFT (Non-Fungible Token).
+    ///
+    /// # Parameters
+    ///
+    /// - `nft_id_ptr`: A pointer to the memory location containing the NFT identifier.
+    /// - `nft_id_len`: The length of the NFT identifier in bytes.
+    /// - `out_buff_ptr`: A mutable pointer to the memory location where the retrieved issuer
+    ///   account will be written.
+    /// - `out_buff_len`: The maximum length of the output buffer.
+    ///
+    /// # Returns
+    ///
+    /// - Returns a positive number of bytes wrote to an output buffer on success
+    /// - Returns a negative error code on failure. The list of error codes is defined in
+    ///   ../core/error_codes.rs
+    pub fn get_nft_issuer(
```

Comment:
I don't know much about the NFT. But does the *NFT* ledger object/data need to be slotted?

Reply:
Only the `get_nft` function uses on-chain data. The `get_nft_...` functions all are just math - processing of the NFT ID to extract information.